app/workers/calculate_bill_worker.rb

```diff
@@ -0,0 +1,36 @@
+class CalculateBillWorker
+  AVERAGE_METER_READING = 20
+  include Sidekiq::Worker
+
+  def perform(*_args)
+    BillingContract.all.each do |billing_contract|
+      delta_meter_reading = get_delta_reading(billing_contract)
+      bill_value = calculate_bill(delta_meter_reading, billing_contract)
+      bill_value = edit_value_depending_payments(bill_value, billing_contract)
+      byebug
+      FactoryBot.create(:bill, amount: bill_value, date: Time.now.utc, billing_contract: billing_contract,
```

Comment:
why do you use here FactoryBot? It is only for seeds and specs? better `billing_contract.bills.create(........`